src/deroff.rs

```diff
@@ -1028,17 +1044,83 @@ impl Deroffer {
         }
     }
 
-    fn do_line(&self, s: &str) -> bool {
-        match s.chars().nth(0) {
-            Some('.') | Some('\'') => !request_or_macro(s),
-            Some(c) => {
+    fn do_tbl(&mut self) -> bool {
+        match self.tblstate {
+            TblState::Options => {
+                while !self.s.is_empty() && !"\n;".contains(self.str_at(0)) {
+                    self.skip_leading_whitespace();
+
+                    if !self.str_at(0).chars().all(|c| c.is_alphabetic()) {
+                        self.skip_char(1);
+                    } else {
+                        // find first non-alphabetic character
+                        match self.s.chars().position(|c| !c.is_alphabetic()) {
+                            Some(idx) if self.s.get(idx..=idx) == Some("(") => {
+                                // self.s -> option '(' arg '(' rest
+                                let option = self.s.split_at(idx).0;
+                                let mut iter = self.s.split_at(idx + 1).1.splitn(2, '(');
```

Comment:
```suggestion
                                let mut iter = self.s[idx + 1..].splitn(2, '(');
```

Reply:
This won't compile if you combined the two together.